libjulia/backends/evm/EVMCodeTransform.cpp

```diff
@@ -187,6 +264,7 @@ void CodeTransform::operator()(assembly::Identifier const& _identifier)
 		}
 	)))
 	{
+		decreaseReference(_identifier.name);
```

Comment:
Please move this to the variable - we do not need to count references to function or labels.